bundles/org.openhab.io.openhabcloud/README.md

```diff
@@ -35,7 +35,7 @@ You will need these values to register on the website before connection is accep
 Location of UUID and Secret:
 
 | File   | Regular Installation                  | APT & RPM Installation               |
-|--------|---------------------------------------|--------------------------------------|
+| ------ | ------------------------------------- | ------------------------------------ |
```

Comment:
Why do you add the spaces there?
I’ve already seen this in a previous PR, but both markdowtable.com and IntelliJ don’t format it that way.

Reply:
I've disabled this rule in my formatter